### PR TITLE
Reload map view annotations in viewWillAppear

### PR DESCRIPTION
**Pull request by kristen**
#91

### CODE REVIEW

File: Mapstagram/ViewControllers/MapViewController.swift [hunk 1]
```diff
@@ -59,6 +59,12 @@ class MapViewController: MediaViewController {
         )
     }
     
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        reloadMediaData()
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/MapViewController.swift]: That's a simple fix!

Why is this necessary exactly?  Somehow the annotations get stale?

Comment (kristen) [inline Mapstagram/ViewControllers/MapViewController.swift]: When I was debugging this I noticed that the reloadMedia for the mapView wasn't getting called when you pull to refresh in the other views (I guess that makes sense since it isn't the current viewController). I then found out that the mapView couldn't find annotationViews for the media objects. So I'm guessing that the images are not removed when you change to a different view controller and when you come back, the view isn't redrawn, it's just left as it was. So I thought of calling reloadMedia in the viewWillAppear and it worked on the first try.

Comment (jamietsao) [inline Mapstagram/ViewControllers/MapViewController.swift]: I see, makes sense.  Merge it!
